operators/api/v1alpha2/instance_types.go

```diff
@@ -31,6 +31,8 @@ const (
 	EnvironmentPhaseImporting EnvironmentPhase = "Importing"
 	// EnvironmentPhaseStarting -> the environment is starting.
 	EnvironmentPhaseStarting EnvironmentPhase = "Starting"
+	// EnvironmentPhaseResourceQuotaExceeded -> the environment could not start because the resource quota are exceeded.
+	EnvironmentPhaseResourceQuotaExceeded EnvironmentPhase = "ResourceQuotaExceeded"
```

Comment:
You need to add this also in the enum above (line 21)

Reply:
Thanks, I will add it in the next commit

Reply:
resolve this conversation when you're done :)